src/run.c

```diff
@@ -37,8 +37,8 @@
 #define UNUSED_PARAMETER(x) (void)(x)
 
 static void usage(const char *cmd) {
-    printf("Usage:  %s [-h] salt [-i|-d|-id] [-t iterations] [-m memory] "
-           "[-p parallelism] [-l hash length] [-e|-r] [-v (10|13)]\n",
+    printf("Usage:  %s [-h] salt [-i|-d|-id] [-t iterations] [-m log2(memory in KiB)] "
+           "[-k memory in KiB] [-p parallelism] [-l hash length] [-e|-r] [-v (10|13)]\n",
```

Comment:
Maybe do something like `[-m log2(memory in KiB) | -k memory in KiB]` to indicate that both flags should not be used at the same time. Also, should there be checking that both `-m` and `-k` are not used?